pages/common/zstdless.md

```diff
@@ -0,0 +1,8 @@
+# zstdless
+
+> View `zstd` compressed files in `less`.
+> More information: <https://manpages.debian.org/testing/zstd/zstdless.1.en.html>.
+
+- Page through a `zstd` compressed file with `less`:
+
+`zstdless {{path/to/file.txt.zst}}`
```

Comment:
This command also works for Arch Linux packages with `.pkg.tar.zst` extension, not only for text compressed files.
```suggestion
`zstdless {{path/to/file.zst}}`
```